components/prow/manifests/nginx-controller.yaml

```diff
@@ -315,31 +315,31 @@ spec:
       - path: /
         backend:
           serviceName: deck
-          servicePort: 80
+          servicePort: http
```

Comment:
Since you are replacing port-name by http, you should also add this as `portName` [deck](https://github.com/prometheus/prombench/blob/a6b972a831f7eb4fe756359fc36ff12502844193/components/prow/manifests/prow_internals_2.yaml#L461) Service object.
The same way it is done in the [hook](https://github.com/prometheus/prombench/blob/a6b972a831f7eb4fe756359fc36ff12502844193/components/prow/manifests/prow_internals_2.yaml#L336) service